patch/msft-kernel-ipv6-ra-metric-sysctl.patch

```diff
@@ -0,0 +1,194 @@
+commit 7c12a9ecf4e0c782875e399ef43220a7fce2f8ca
+Author: pchaudha <pchaudha@contoso.com>
```

Comment:
Does this email address work? [Contoso is a fictional company used in Microsoft documentation(?).](https://de.wikipedia.org/wiki/Contoso)

Reply:
No Comment abt it ;)